internal/cloudsql/instance.go

```diff
@@ -171,7 +171,10 @@ type ConnectionInfo struct {
 	ConnectionName    instance.ConnName
 	ClientCertificate tls.Certificate
 	ServerCaCert      *x509.Certificate
+	ServerCaCertPem   []byte // Only set for CAS instances.
+	ServerCaMode      string // Only set for CAS instances.
 	DBVersion         string
+	DNSName           string // Only set for CAS instances.
```

Comment:
`DomainName` (not DNSName)

Reply:
I am not sure about this. Because this `DNSName` is directly from the `dnsName` of the `ConnectSettings` API [here](https://cloud.google.com/sql/docs/mysql/admin-api/rest/v1beta4/connect/get#response-body).

Reply:
Probably better to stick with the wording from the ConnectSettings API. We could add a godoc comment explaining what this is used for though.